chain-storage/src/buffer.rs

```diff
@@ -318,6 +319,22 @@ pub fn flush_kvdb<S: KeyValueDB + ?Sized, H: BuildHasher>(
     storage.write(tx)
 }
 
+/// Flush buffer to storage
+pub fn flush_storage<H: BuildHasher>(
+    storage: &mut Storage,
+    buffer: HashMap<(u32, Vec<u8>), Option<Vec<u8>>, H>,
+) -> std::io::Result<()> {
```

Comment:
`use std::io::Result`

Reply:
That would conflict with prelude `Result`, then you need to rename into `IoResult` or something, I think the fully qualified name is short enough to use directly?